tests/ui/pages/desktop/home.py

```diff
@@ -39,3 +52,106 @@ def users(self):
                 """Number of users that have downloaded the extension"""
                 users_str = self.find_element(*self._users_locator).text
                 return int(users_str.split()[0].replace(',', ''))
+
+    class FeaturedExtensions(Region):
+        """ Featured Extension region """
+
+        _root_locator = (By.ID, 'featured-extensions')
+
+        _extension_locator = (By.CSS_SELECTOR, 'section > li > div')
+        _see_all_locator = (By.CSS_SELECTOR, 'h2 > a')
+
+        @property
+        def extensions(self):
+            return [self.Extension(self.page, el) for el in self.find_elements(
+                *self._extension_locator)]
+
+        class Extension(Region):
+
+            _name_locator = (By.CSS_SELECTOR, 'h3')
+            _link_locator = (By.TAG_NAME, 'a')
+
+            def see_all(self):
+                """
+                Clicks the 'See All' link within the Featured Extensions
+                region.
+                """
+                self.find_element(*self._link_locator).click()
+                from pages.desktop.details import Details
```

Comment:
why is it not being used? Technically imports are only executed once so this can easily be at the top anyway. But I don't care too much, if there's a strong argument for it to stay this way I don't mind, if not, I'd rather have imports at the top since that's where they belong.